google/cloud/forseti/common/gcp_api/appengine.py

```diff
@@ -430,6 +430,9 @@ def list_instances(self, project_id, service_id, version_id):
                          project_id, service_id, version_id, flattened_results)
             return flattened_results
         except (errors.HttpError, HttpLib2Error) as e:
+            if e.args[0].status == 501:  # pylint: disable=no-member
```

Comment:
Is this the right way of checking status code? I thought you can access the status by doing this `e.resp.status == 501`. Can we also add a test case for this

Reply:
Here is an example on how we did checks similar to this before:
https://github.com/forseti-security/forseti-security/blob/3e489059bc5b3049e6671a8e45cb32b9bd0730ea/google/cloud/forseti/common/gcp_api/compute.py#L47

Reply:
Done on the `e.resp.status`, good tip

Reply:
Hmmm, in that example, `403` can be very broad, and is giving the ability to check for specific errors and handle it differently.  But in this case of `501`, I don't see how many different ways this needs to be handled.